vision_agent/tools/tools.py

```diff
@@ -2142,7 +2180,7 @@ def overlay_bounding_boxes(
         bboxes = bbox_int[i]
         bboxes = sorted(bboxes, key=lambda x: x["label"], reverse=True)
 
-        if len(bboxes) > 40:
+        if len(bboxes) > 100:
```

Comment:
@dillonalaird what is this magic number used for? Can you please add a small comment describing its purpose?

Reply:
Good point! Added some comments here